.github/workflows/main.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         password: ${{ secrets.GADOCKERSVC_PASSWORD }}
 
     - name: Build Docker
-      if: steps.changes.outputs.docker == 'true'
+      # if: steps.changes.outputs.docker == 'true'
```

Comment:
I think we want to leave this `if` statement. It's there to prevent rebuilding the docker-test image on every GHA run. With the `if` it only reruns if something has changed in the `./docker/` directory.

Reply:
Yeah, I was thinking the same thing - I left it in in #1433

Reply:
Actually, I think it DOES need to be commented out - because the "latest" Docker image on dockerhub will be build with SQLAlchemy1.4, we ALWAYS need build on this branch.